core/math/quick_hull.cpp

```diff
@@ -376,10 +375,20 @@ Error QuickHull::build(const Vector<Vector3> &p_points, Geometry::MeshData &r_me
 
 			Map<Edge, RetFaceConnect>::Element *F = ret_edges.find(e);
 
-			ERR_CONTINUE(!F);
+			if (!F) {
+				warning_f = true;
+				continue;
+			}
```

Comment:
`ERR_*` macros use `unlikely()` for the condition which *might* improve branch prediction. So by redoing the logic without `unlikely()` here it *might* actually impact performance.